package/MDAnalysis/analysis/msd.py

```diff
@@ -267,21 +268,34 @@
 class EinsteinMSD(AnalysisBase):
     r"""Class to calculate Mean Squared Displacement by the Einstein relation.
 
+    If `fft=False` so that the "windowed" algorithm is used, the second order
+    nongaussian parameter can also computed with ``nongaussian==True`` to 
+    capture heterogeneity as different time scales. See :cite:p:`Fuchs1998` 
+    for more information. 
+
+    .. math::
+    
+       \alpha_{2}(r_{d}) = \frac{3}{5} \frac{ \bigg{\langle} \frac{1}{N} \sum_{i=1}^{N} |r_{d}
+        - r_{d}(t_0)|^4 \bigg{\rangle} } { \bigg{\langle} \frac{1}{N} \sum_{i=1}^{N} |r_{d}
+        - r_{d}(t_0)|^2 \bigg{\rangle}^2 } - 1
+
     Parameters
     ----------
     u : Universe or AtomGroup
         An MDAnalysis :class:`Universe` or :class:`AtomGroup`.
         Note that :class:`UpdatingAtomGroup` instances are not accepted.
-    select : str
+    select : str (optional)
         A selection string. Defaults to "all" in which case
         all atoms are selected.
-    msd_type : {'xyz', 'xy', 'yz', 'xz', 'x', 'y', 'z'}
+    msd_type : {'xyz', 'xy', 'yz', 'xz', 'x', 'y', 'z'} (optional)
         Desired dimensions to be included in the MSD. Defaults to 'xyz'.
-    fft : bool
+    fft : bool (optional)
         If ``True``, uses a fast FFT based algorithm for computation of
         the MSD. Otherwise, use the simple "windowed" algorithm.
         The tidynamics package is required for `fft=True`.
         Defaults to ``True``.
+    nongaussian : bool (optional)
```

Comment:
Nitpick, idk what other people would say but personally I wouldn't say optional because has a default and will raise exception otherwise.

Reply:
If `fft==False` then this is optional and won't raise an exception. If can be `True` or `False`